prime/src/components/borrow/actions/UniswapAddLiquidityActionCard.tsx

```diff
@@ -305,9 +309,25 @@ export default function UniswapAddLiquidityActionCard(props: ActionCardProps) {
           priceToTick(price, token0.decimals, token1.decimals),
           tickInfo.tickSpacing
         );
+        const priceAtNearestTick = numericPriceRatioGN(
+          tickToPrice(nearestTick),
+          token0.decimals,
+          token1.decimals,
+          !isToken0Selected
+        );
+        const priceAboveNearestTick = numericPriceRatioGN(
+          tickToPrice(nearestTick + tickInfo.tickSpacing),
+          token0.decimals,
+          token1.decimals,
+          !isToken0Selected
+        );
+        const closestTick =
+          Math.abs(price - priceAtNearestTick) < Math.abs(price - priceAboveNearestTick)
+            ? nearestTick
+            : nearestTick + tickInfo.tickSpacing;
```

Comment:
I don't understand what this is for. We already define `nearestTick` -- why is `closestTick` necessary on top of that? If anything, the functions we use to compute `nearestTick` should be fixed, instead of adding this extra logic here.

Reply:
Could you take a stab at fixing the underlying function? I spent a bit of time playing with it to no avail.

Reply:
Definitely. Can you describe what's wrong with it so I know what to fix?